src/mmw/apps/user/views.py

```diff
@@ -81,10 +82,14 @@ def logout(request):
         return redirect('/')
 
 itsi = ItsiService()
+ITSI_URL_FLAG = settings.ITSI['url_flag']
 
 
 def itsi_login(request):
-    redirect_uri = request.build_absolute_uri(reverse('itsi_auth'))
+    redirect_uri = '{0}?next={1}'.format(
```

Comment:
Either is fine, I prefer `.format` slightly more. Those Python guidelines are not authored by anyone on the Civic Apps team and could probably be updated.